.github/workflows/release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - uses: goreleaser/goreleaser-action@v2
         with:
           version: latest
-          args: release --rm-dist
+          args: release --rm-dist -ldflags="-X 'terraform-provider-iterative/iterative/utils.Version=${GITHUB_REF##refs/tags/v}'"
```

Comment:
@0x2b3bfa0 this should set the Version var in the utils library to be  used in `tool_version`

Reply:
Yup

Reply:
I think its cleaner `${{ github.ref_name}}` vs `${GITHUB_REF##refs/tags/v}`

Reply:
#537